backend/inspirehep/records/cli.py

```diff
@@ -178,3 +182,49 @@ def _finished_tasks_count():
         LOGGER.warning(f"Got {len(failures)} failures during the updating process")
         for failure in failures:
             LOGGER.warning(failure)
+
+
+@click.group()
+def jobs():
+    """Command for jobs"""
+
+
+@jobs.command(help="Closes jobs with deadline before the given date.")
+@click.option(
+    "--deadline-before-date",
+    help="Date with the format YYYY-MM-DD. All jobs with deadline date before this date "
+    "will be closed.",
+    type=str,
+    default=str(datetime.datetime.utcnow().date()),
+    show_default=True,
+)
+@with_appcontext
+def close_before(deadline_before_date):
+    datetime.datetime.strptime(deadline_before_date, "%Y-%m-%d")
+
+    before_deadline_date = (
+        type_coerce(RecordMetadata.json, JSONB)["deadline_date"].astext.cast(DateTime)
+        < deadline_before_date
+    )
+    only_jobs_collection = type_coerce(RecordMetadata.json, JSONB)[
+        "_collections"
+    ].contains(["Jobs"])
+    only_not_closed = not_(
+        type_coerce(RecordMetadata.json, JSONB)["status"].astext == "closed"
+    )
+    only_not_deleted = or_(
+        not_(type_coerce(RecordMetadata.json, JSONB).has_key("deleted")),
+        not_(RecordMetadata.json["deleted"] == cast(True, JSONB)),
+    )
+    expired_jobs = RecordMetadata.query.filter(
+        only_jobs_collection, only_not_deleted, only_not_closed, before_deadline_date
+    ).all()
+
+    with db.session.begin_nested():
+        for job in expired_jobs:
+            record = JobsRecord(job.json, model=job)
+            record["status"] = "closed"
+            record.update(dict(record))
+    db.session.commit()
+    click.echo(f"Closed {len(expired_jobs)} jobs")
+    LOGGER.info(f"Closed {len(expired_jobs)} jobs")
```

Comment:
What's the point of both logging and printing? the logger already prints on stdout.
```suggestion
    LOGGER.info("Closed outdated jobs", num_records=len(expired_jobs))
```